src/cmd/sign.js

```diff
@@ -125,6 +156,170 @@ export default function sign(
   );
 }
 
+/**
+ * Fetches an updateManifest.json file from a remote web server
+ */
+function fetchUpdateManifest(manifestData: Object): Object {
```

Comment:
The Flow return value should be `Promise<Object>` but it would be better if you could define the parts of the manifest object you'll need and specify a type for it. See this example: https://github.com/mozilla/web-ext/blob/master/src/util/manifest.js#L14-L18

After a custom definition, the return value would become `Promise<UpdateManifest>`.